docs/pages/management/operations/scaling.mdx

```diff
@@ -124,4 +124,37 @@ has default limits of 4CPU and 4Gi memory.
 | SSH           | 1000     | Auth CPU Limits exceeded  |
 | Application   | 2500     | Proxy CPU Limits exceeded |
 | Database      | 40       | Proxy CPU Limits exceeded |
-| Kubernetes    | 50       | Proxy CPU Limits exceeded |
\ No newline at end of file
+| Kubernetes    | 50       | Proxy CPU Limits exceeded |
+
+## Windows Desktop Service
+
+Windows Desktop sessions can vary greatly in resource usage depending on the applications being used. The primary
+factor affecting resource usage per session is how often the screen is updated. For example, a session playing a video
+in full screen mode will consume significantly more resources than a session where the user is typing in a text editor.
+
+We measured the resource usage of sessions playing fullscreen videos to get the worst-case estimate for resource requirements.
+We then inferred resource requirements for more standard use cases on the basis of those measurements.
+
+**Worst Case:**
+- 1/12 vCPU per concurrent session
```

Comment:
Consider using rounder number here, e.g. `1/10`, `50MB`, `1/200`

Reply:
`1/12` is more accurate. It also feels "round-ish" to my imperial system sensibilities :)